Makefile

```diff
@@ -326,23 +326,20 @@ benchmark:
 ###############################################################################
 
 golangci_lint_cmd=golangci-lint
-golangci_version=v1.56.2
+golangci_version=v1.61.0
 
 lint:
 	@echo "--> Running linter"
-	@go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(golangci_version)
```

Comment:
This results in an error:

```
$ make lint
--> Running linter
INFO golangci-lint has version v1.61.0 built with go1.22.6 from (unknown, modified: ?, mod sum: "h1:VvbOLaRVWmyxCnUIMTbf1kDsaJbTzH20FAMXTAlQGu8=") on (unknown)
INFO [config_reader] Config search paths: [./ /Users/rootulp/git/rootulp/celestiaorg/cosmos-sdk /Users/rootulp/git/rootulp/celestiaorg /Users/rootulp/git/rootulp /Users/rootulp/git /Users/rootulp /Users /]
INFO [config_reader] Used config file .golangci.yml
Error: can't load config: the Go language version (go1.22) used to build golangci-lint is lower than the targeted Go version (1.23.1)
Failed executing command with error: can't load config: the Go language version (go1.22) used to build golangci-lint is lower than the targeted Go version (1.23.1)
make: *** [lint] Error 3
```

The fix is to use the version of golangci-lint installed on the machine (should be v1.61.0)